File: phoenixfront/src/app/models/archive/ArchivedProductsDTO.ts
export interface ArchivedProductsDTO {
    serialNumber?: string;
    simNumber?: string;
    checkin?: Date;
    checkout?: Date;
    boxNumber?: string;
    productType?: string;
    brand?: string;
    prodName?: string;
    comments?: string;
    price?: number;
    returned?: boolean;
    returnedstatus?: boolean;
    checkedExistence?: boolean;
    stockReference?: string;
}

File: phoenix/stock/src/main/java/com/phoenix/archivedmodeldto/ArchivedProductsDTO.java
package com.phoenix.archivedmodeldto;

import lombok.*;

import java.math.BigDecimal;
import java.time.LocalDate;
@Data
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class ArchivedProductsDTO {
    private String serialNumber;
    private String simNumber;
    private LocalDate checkin;
    private LocalDate checkout;
    private String boxNumber;
    private String productType;
    private String brand;
    private String prodName;
    private String comments;
    private BigDecimal price;
    private boolean returned;
    private boolean returnedstatus;
    private boolean checkedExistence;
    private String stockReference;
}
